### DISCUSSION TITLE
Button Interaction

### DISCUSSION THREAD

**Discussion by toastyy00** [rigwild/discord-self-bot-console#67] [Q&A]
how to press the button/components or each button on a specific discord message id like this
![image](https://github.com/rigwild/discord-self-bot-console/assets/108380151/508f19ce-a574-43a1-b01a-63cc92916664)

**Comment by DeadLyBro** [comment]
Did you tried https://github.com/rigwild/discord-self-bot-console/discussions/55#discussioncomment-5608127 ?
